UNLOCK.md

```diff
@@ -0,0 +1,40 @@
+# Unlock with UIAutomation (New)
```

Comment:
would prefer to see this in `docs/unlock.md` (new dir, no caps). looks great though!